games/games.js

```diff
@@ -83,28 +93,46 @@ logoutButton.addEventListener('click', () => {
  // on load . . .
 window.addEventListener('', async() => {
```

Comment:
need an action inside the empty strings here on line 94 so when we load it will actually run this function, currently it doesn't run your `displayAllGames()` call